src/commands/mod.rs

```diff
@@ -7,9 +7,13 @@ use std::fs;
 use std::fs::File;
 use std::io::Write;
 use std::path::Path;
+use std::str::FromStr;
 use std::time::Duration;
 
 use serde::{Deserialize, Deserializer, Serialize};
+use serde_json::json;
+use strum::IntoEnumIterator;
```

Comment:
I don't see this one used.

Reply:
`use serde_json::json;` -> is used for ```.filter_map(|(key, value)| value.as_ref().map(|v| (*key, json!(v))))``` to omit None fields in the payload
`use strum::IntoEnumIterator;` -> is used for ```let valid_setting_types = SettingType::iter()```